src/test/resources/com/puppycrawl/tools/checkstyle/config-custom-initial-module.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
+
+<module name="com.puppycrawl.tools.checkstyle.TestInitialModuleChecker">
+</module>
```

Comment:
just add new line to end of file, as you will change PR to address other points.